plugin/src/test/java/io/jenkins/plugins/casc/impl/configurators/DataBoundConfiguratorTest.java

```diff
@@ -104,6 +124,9 @@ public Foo(String foo, boolean bar, int qix) {
             this.foo = foo;
             this.bar = bar;
             this.qix = qix;
+            if (qix == 99) {
+                throw new RuntimeException("Magic test fail");
```

Comment:
codacy is currently failing because of this,
maybe:
```suggestion
                throw new IllegalArgumentException("Magic test fail");
```